.travis.yml

```diff
@@ -1,11 +1,23 @@
-sudo: false
-dist: trusty
-language: python
-python:
-  - 3.6
+sudo: required
```

Comment:
Previously we ran inside a container, now we run on a VM.
It is important to understand that this may impact the resources we get from the CI. We should monitor this.

Please mention this detail in the commit message.